tests/test_pattern.py

```diff
@@ -443,10 +446,10 @@ def test_pauli_measurement_then_standardize(
     @pytest.mark.parametrize("jumps", range(1, 11))
     def test_standardize_two_cliffords(self, fx_bg: PCG64, jumps: int) -> None:
         rng = Generator(fx_bg.jumped(jumps))
-        c0, c1 = rng.integers(len(clifford.TABLE), size=2)
+        c0, c1 = rng.integers(len(CTAB), size=2)
         pattern = Pattern(input_nodes=[0])
-        pattern.add(C(node=0, clifford=clifford.TABLE[c0]))
-        pattern.add(C(node=0, clifford=clifford.TABLE[c1]))
+        pattern.add(C(node=0, clifford=CTAB[c0]))
```

Comment:
It would be better to be able to do `Clifford(c0)`, without worrying about having a new instance.

Reply:
I totally agree, but then it would be better implemented using the singleton pattern...

Reply:
Resolved by `Enum` inheritance.